mobile/src/main/java/rs/readahead/washington/mobile/views/dialog/TellaUploadServerDialogFragment.java

```diff
@@ -303,8 +303,8 @@ private void checkServer(@NonNull TellaUploadServer server,  boolean connectionR
                 return;
```

Comment:
we are not using this anymore?

Reply:
It's part of Google Mobile Services so we can't use it. But yes, we should think about some replacement for FOSS.